MSTR-120: Render markdown back to HTML (#544)
2600hz/monster-ui: docs/monster-ui/markdownToHtml().md

title: markdownToHtml()

# monster.ui.markdownToHtml()

> The `monster.ui.markdownToHtml` render markdown content back to HTML.

Reviewer: ```suggestion
The `monster.ui.markdownToHtml` renders markdown content as HTML.
```

## Syntax
```javascript
monster.ui.markdownToHtml(content);
```

### Parameters
Key | Description | Type | Default | Required
:-: | --- | :-: | :-: | :-:

> `content` | Markdown content that needs to be rendered to HTML | `String` | | `true`

Reviewer: ```suggestion
`content` | Markdown content to be rendered as HTML | `String` | | `true`
```

## Description

> This helper will allows yo to render markdown content to HTML

Reviewer: ```suggestion
This helper allows you to render markdown content as HTML
```

## Example
```javascript
var markdownContent = '**Hello world**',

>   description = monster.ui.markdownToHtml(markdownContent);

Reviewer: ```suggestion
    description = monster.ui.markdownToHtml(markdownContent);
```

>   template.find('.description').html(description);

Reviewer: ```suggestion
template.find('.description').html(description);
```
